ykbh/src/lib.rs

```diff
@@ -0,0 +1,310 @@
+#![feature(test)]
+extern crate test;
```

Comment:
Do we need these two lines? I vaguely thought they weren't needed in the 2018 edition?

Reply:
Will try.